src/Microsoft.ML.AutoML/Experiment/OptimizingMetricInfo.cs

```diff
@@ -40,5 +44,10 @@ public OptimizingMetricInfo(MulticlassClassificationMetric multiMetric)
         {
             IsMaximizing = !_minimizingMulticlassMetrics.Contains(multiMetric);
         }
+
+        public OptimizingMetricInfo(RankingMetric rankingMetric)
+        {
+            IsMaximizing = !_minimizingRankingMetrics.Contains(rankingMetric);
```

Comment:
This is always `True`, maybe just
`IsMaximizing=True;` ?

Reply:
Yes, it is always true for NDCG and DCG.